kudo-operator/templates/spark-history-server-deployment.yaml

```diff
@@ -0,0 +1,53 @@
+{{ if .Params.enableHistoryServer }}
```

Comment:
Let's use hyphens when we template conditionals, we use it throughout the rest of the templates and IMO it improves readability and allows distinguishing control structures from variables:
```
{{- if .Params.enableHistoryServer }}
<spec>
{{- end }}
```